pyslackersweb/cli/migrate.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
We probably need to throw some segment of these into a [transaction](https://magicstack.github.io/asyncpg/current/usage.html#transactions) to ensure they run _all or nothing_. At a minimum that should probably be both lines L#26 and L#27-29 (or just nest L#37)

Reply:
I was sure it was doing an automatic transaction behind the scene, but doesn't look like it in the end. I must have mixed that with another library I was looking at.